prepare-incremental-build/action.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Prepare incremental kernel build'
+description: 'Pull cached kernel build output from previous runs and prepare the repo for incremental build'
+inputs:
+  repo-root:
+    description: "Root of the kernel repository"
+    required: true
+    default: ${{ github.workspace }}
```

Comment:
in my experience, this does not work. We have it in a few places, but I don't think it works.

It may be worth testing separately through a simple action.

Based on https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#context-availability , it is not mentioned that any contexts are available in the "inputs" part of an action. (It would work for a reusable workflow though).

Reply:
replying to myself... on the other hand, there is things like https://github.com/actions/checkout/blob/6d193bf28034eafb982f37bd894289fe649468fc/action.yml#L6C5-L6C38 which suggests it should work, so mayke I did something wrong when testing.

Reply:
Yes, it's not clear from the docs. I can double check, although in this case I think we can just remove the default and require this input to be specified.

Reply:
Done.